sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -22,86 +22,91 @@
 GCS_STAGING_BLOB_DEFAULT_PREFIX = 'kfp_container_build_staging'
 GCR_DEFAULT_IMAGE_SUFFIX = 'kfp_container'
 
+
+def _get_project_id():
+  import requests
+  URL = "http://metadata.google.internal/computeMetadata/v1/project/project-id"
+  headers = {
+    'Metadata-Flavor': 'Google'
+  }
+  r = requests.get(url = URL, headers = headers)
+  if not r.ok:
+    raise RuntimeError('ContainerBuilder failed to retrieve the project id.')
+  return r.text
+
+
+def _get_instance_id():
+  import requests
+  URL = "http://metadata.google.internal/computeMetadata/v1/instance/id"
+  headers = {
+    'Metadata-Flavor': 'Google'
+  }
+  r = requests.get(url = URL, headers = headers)
+  if not r.ok:
+    raise RuntimeError('ContainerBuilder failed to retrieve the instance id.')
+  return r.text
+
+
 class ContainerBuilder(object):
   """
   ContainerBuilder helps build a container image
   """
-  def __init__(self, gcs_staging=None, gcr_image_tag=None, namespace=None):
+  def __init__(self, gcs_staging=None, default_image_name=None, namespace=None):
     """
     Args:
       gcs_staging (str): GCS bucket/blob that can store temporary build files,
           default is gs://PROJECT_ID/kfp_container_build_staging.
-      gcr_image_tag (str): GCR image tag where the target image is pushed
+      default_image_name (str): GCR image tag where the target image is pushed
```

Comment:
`default_image_name (str): GCR image tag where the target image is pushed` Does the description need to update, too?

Reply:
Thanks for the catch. Fixed.